rootless.md

```diff
@@ -0,0 +1,31 @@
+# Shortcomings of Rootless Podman
+
+* Lacking “How To” documentation or documentation in general
+* If /etc/subuid and /etc/subgid not setup for a user, then podman commands
+can easily fail
+  * This can be a big issue on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
+  * We are working to get support for NSSWITCH on the /etc/subuid and /etc/subgid files.
+* Podman can not create containers that bind to ports < 1024.
+  * The kernel does not allow processes without CAP_NET_BIND_SERVICE to bind to low ports.
+* No cgroup Support (hopefully fixed when cgroups V2 happens).
+  * Cgroups V1 does ont support cgroup delegation.
+* Can not share container images with CRI-O or other users
+* Difficult to use additional stores for sharing content
+* Does not work on NFS homedirs
+* Does not work with homedirs mounted with noexec/nodev
+* Can not use overlayfs driver, but does support fuse-overlayfs
+  * Does not perform as well as root podman with Kernel based Overlay
```

Comment:
I assumed so, but doing some performance testing, I cannot see any significant differences in the two implementations (in some tests fuse-overlayfs is even faster).

Reply:
On Ubuntu overlayfs should work

Reply:
I think we can remove the lines about performance, then.   
@giuseppe If I don't install fuse-overlay on an Ubuntu host do we use overaly?

Reply:
no, we default to vfs, I've not really tried to use overlay on Ubuntu but it can be manually configured

Reply:
I helped someone set it up in IRC, and it seemed properly simple (remove the mount-program line from the config). Not sure if there's an easy way to probe the kernel to see that unprivileged overlay is available, though.